helper/logging/logfile.go

```diff
@@ -0,0 +1,74 @@
+package logging
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+)
+
+var now = time.Now
+
+type LogFile struct {
+	// Name of the log file
+	fileName string
+
+	// Path to the log file
+	logPath string
+
+	// LastCreated represents the creation time of the latest log
+	LastCreated time.Time
+
+	// FileInfo is the pointer to the current file being written to
+	FileInfo *os.File
+
+	// BytesWritten is the number of bytes written in the current log file
+	BytesWritten int64
```

Comment:
I assume this is part of logfile rotation, but you don't seem to have ported the rotation parts from Consul?

Reply:
I didn't plan on bring the rotation along for the ride. :( 

`FileInfo` is the pointer to the file, which if we don't have .. we create a new file, so I believe that's relevant.

`BytesWritten` we could ditch for sure, I think I pondered leaving it in at one point, then forgot to take it out. Doh.